pkg/components/exec_node.go

```diff
@@ -14,21 +14,22 @@ import (
 )
 
 type ExecNode struct {
+	localServerComponent
 	baseExecNode
-	localComponent
 	master Component
 }
```

Comment:
Do you think it is ok to have server twice: inside localServerComponent struct and inside baseExecNode here?

Reply:
There is no problems with that since this is reference, not instance itself.